Makefile

```diff
@@ -35,7 +35,13 @@ install-addlicense:
 install-lint:
 	(which $(GOPATH)/bin/golangci-lint || go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.40.1)
 
-generate: install-stringer
+install-deepcopy-gen:
+	(which $(GOPATH)/bin/deepcopy-gen || go install k8s.io/code-generator/cmd/deepcopy-gen@v0.23.3)
+
+generate-deepcopy: install-deepcopy-gen
+	hack/run-in-gopath.sh deepcopy-gen --input-dirs ./pkg/apis/... -O zz_generated.deepcopy --go-header-file ./LICENSE_TEMPLATE_GO
```

Comment:
`run-in-gopath.sh` is used only in `make generate-deepcopy`. Shall we also update other targets?

Reply:
Nah. Only deepcopy-gen needs it. Everything else works in Go's module mode.
Upstream Kubernetes does the same thing for its generators.

Reply:
Ack